recipes-bsp/xt-rpi-u-boot-scr/files/boot.cmd.xen.in

```diff
@@ -23,8 +23,5 @@ fdt set /chosen/domD/module@@@DOMD_IMG_ADDR@@ reg <0x0 @@DOMD_IMG_ADDR@@ 0x0 0x$
 fdt mknod /chosen/domD module@@@DOMD_DTB_ADDR@@
 fdt set /chosen/domD/module@@@DOMD_DTB_ADDR@@ compatible  "multiboot,device-tree" "multiboot,module"
 fatload @@BOOT_MEDIA@@ 0 @@DOMD_DTB_ADDR@@ @@DOMD_DTB@@
-fdt set /chosen/domD/module@@@DOMD_DTB_ADDR@@ reg <0x0 @@DOMD_DTB_ADDR@@ 0x0 0x${filesize} >
-
-setenv fdt_high 0xffffffffffffffff
-
-booti @@XEN_IMG_ADDR@@ - ${fdt_addr}
+fdt set /chosen/domD/module@@@DOMD_DTB_ADDR@@ reg <0x0 @@DOMD_DTB_ADDR@@ 0x0 0x6000 >
```

Comment:
why did you replaced ${filesize} with 0x6000?

Reply:
Because we modify loaded DT and size if DT changed. There is no way to calculate accurate DT size